src/app/containers/Bulletin/index.stories.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import pathOr from 'ramda/src/pathOr';
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+import { RequestContextProvider } from '#contexts/RequestContext';
+import BulletinContainer from '.';
+import fixture from '#data/igbo/frontpage';
+import AmpDecorator from '../../../../.storybook/helpers/ampDecorator';
+
+const bulletinFixture = type =>
+  pathOr(null, ['content', 'groups'], fixture)
+    .flatMap(group => pathOr(null, ['items'], group))
+    .find(
+      item =>
+        pathOr(null, ['assetTypeCode'], item) === 'PRO' &&
+        pathOr(null, ['contentType'], item) === type,
```

Comment:
I don't know how often this fixture data changes, but this will return the first bulletin, whether it is live or not.
```suggestion
        pathOr(null, ['contentType'], item) === type &&
        !pathOr(null, ['isLive'], item) === true,
```
would ensure it would only select a non-live bulletin. This may be overkill though depending on how often fixtures change. Could even have this as a small seperate ticket if we need this to move a bit faster

Reply:
Good spot! I think just `!pathOr(null, ['isLive'], item)`, should be enough?

Reply:
Yep, that should be enough!